cli/lib/tasks/verify.js

```diff
@@ -258,8 +258,13 @@ const start = (options = {}) => {
     force: false,
     welcomeMessage: true,
     smokeTestTimeout: VERIFY_TEST_RUNNER_TIMEOUT_MS,
+    skipVerify: util.getEnv('CYPRESS_SKIP_VERIFY') || 'false',
```

Comment:
Can we get this to just work with booleans?
```suggestion
    skipVerify: util.getEnv('CYPRESS_SKIP_VERIFY') === 'true',
```